### PR TITLE
Updated direct connection config docs

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated direct connection config docs
Closes https://github.com/Azure/azure-sdk-for-java/issues/18094

### CODE REVIEW

**Review by simplynaveen20 — APPROVED** [review]

**Review by xinlian12 — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ public DirectConnectionConfig setIdleEndpointTimeout(Duration idleEndpointTimeou
      * Gets the max connections per endpoint
      * This represents the size of connection pool for a specific endpoint
      *
-     * Default value is 30
+     * Default value is 130
```
Comment (simplynaveen20) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java:183]: Can we use the variable in the doc ?

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java (hunk 1 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java:183]:
```suggestion
     * Default value is {@value #DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT}
```

**Review by simplynaveen20 — CHANGES_REQUESTED** [review]
Please use variable in the doc

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java:183]: Tried this, but didn't work. 
Tried this as well, didn't work either: 
```
Default value is {@value com.azure.cosmos.DirectConnectionConfig#DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT}
```

**Review by simplynaveen20 — APPROVED** [review]
As maven giving warning while using variable in doc , approving the PR . LGTM . Thanks @kushagraThapar

**Pull request merged**